README.md

```diff
@@ -104,20 +106,76 @@ buildscript {
 }
 ```
 
+</details>
+
+<details><summary>Groovy</summary>
+
+```groovy
+buildscript {
+    repositories {
+        // ...
+        jcenter()
+    }
+
+    dependencies {
+        // ...
+        classpath 'com.github.triplet.gradle:play-publisher:2.0.0-beta2'
+    }
+}
+```
+
+</details>
+</br>
+
 Then apply it to each individual `com.android.application` module where you want to use GPP.
 For example, `app/build.gradle(.kts)` is a commonly used app module:
 
+<details open><summary>Kotlin</summary>
+
+```kt
+plugins {
+    id("com.android.application")
+    id("com.github.triplet.play")
+}
+```
+
+</details>
+
+<details><summary>Groovy</summary>
+
 ```groovy
 apply plugin: 'com.android.application'
 apply plugin: 'com.github.triplet.play'
 ```
 
+</details>
+
 #### Snapshot builds
 
 If you're prepared to cut yourself on the bleeding edge of GPP development, snapshot builds are
 available from
 [Sonatype's `snapshots` repository](https://oss.sonatype.org/content/repositories/snapshots/):
 
+<details open><summary>Kotlin</summary>
+
+```kt
+buildscript {
+    repositories {
+        // ...
+        maven { url = uri("https://oss.sonatype.org/content/repositories/snapshots") }
```

Comment:
Can't we use `maven("https://oss.sonatype.org/content/repositories/snapshots")` here?

Reply:
Didn't know that was a thing, awesome!